### DISCUSSION TITLE
Using docker-slim for dedicated gaming server image

### DISCUSSION THREAD

**Discussion by thasmo** [slimtoolkit/slim#185] [Q&A]
👋 Would `docker-slim` also work for dedicated gaming server images or is it intended to optimize docker images running more classic web/software applications only?

**Comment by kcq** [comment]
@thasmo yes, it is possible. 

It might require a bit more work configuring additional parameters to docker-slim. I assume there's more than one application in your gaming server container images. Some of them might be server apps while others are CLIs. Making sure that all of those apps get to run when docker-slim is executing is the first step.

Do you mind sharing a bit more about your container images and how they are structured? Happy to help anyway I can.

**Reply by thasmo** [reply]
@kcq 👋 Thanks! It's a [Docker image for `Don't Starve Together`](https://github.com/yeetzone/docker-dontstarvetogether) installed via `steamcmd`. I did not have a look at the `docker-slim` docs yet, wanted to know if it would actually be possible in the first place.

**Reply by kcq** [reply]
That's great that it's available online. Overall looks possible. The entrypoint calls a number of scripts and then the server runs. Would be interesting to try. It does look like I need a special token for it. Would it be possible to generate a temp token for testing?

**Reply by thasmo** [reply]
True. I've created `pds-g^KU_pM4mebfa^Im+ZJ9Ra3+GuMyhN8IitWnWU5n9Jx6HOjKYAs4SoD9w=` as "temporary" token for you to try, which I will delete at a later time.

**Reply by kcq** [reply]
Thanks @thasmo ! I'll post my findings and i'll bug you with more questions if I get stuck :-)

**Reply by thasmo** [reply]
FYI; I've revoked the token for now, after I realized, at this point in time, that I had no time to have a look at `docker-slim` anyway. 😅
